frontend/src/app/application-forms/fields/quantity.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { ApplicationFieldsService } from '../_services/application-fields.service';
+
+@Component({
+  selector: 'app-quantity',
+  templateUrl: './quantity.component.html'
+})
+export class QuantityComponent {
+  @Input() parentGroup: FormGroup;
+  @Input() label: string;
+  @Input() hintText: string;
```

Comment:
In looking at a lot of the other field components, length & labels are not included. Is that how we are moving to validation gong forward?